cmd/convox/run.go

```diff
@@ -40,7 +40,6 @@ func cmdRun(c *cli.Context) {
 
 	fd := os.Stdin.Fd()
 	stdinState, err := terminal.GetState(int(fd))
-	defer terminal.Restore(int(fd), stdinState)
```

Comment:
ah, got it.  (couldn't figure out what that was for!)  I'll add it back in and put the conditional around it.  and a comment ;)